nixos/modules/misc/locate.nix

```diff
@@ -230,9 +230,9 @@ in
         plocate = (mkIf isPLocate (mkMerge [ common plocate ]));
       };
 
-    environment.systemPackages = [ (cfg.locate.override { dbfile = cfg.output; }) ];
```

Comment:
Now the option does not really work anymore. A more complete fix is/was available at #258434 which I am now going to rebase.

Reply:
I've checked the source code for both mlocate and plocate, and both respect `LOCATE_PATH` now, so it should work correctly in all cases without the override.

Reply:
Not that I'm opposed to removing the option.

Reply:
The compile options for mlocate/plocate need to be reverted otherwise it points by default to the nix store or /var/lib/

Reply:
I've pushed the changes I like to see to #258434